src/components/store/statistics/Era.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div
+    class="
+      tw-bg-white
+      dark:tw-bg-darkGray-800
+      tw-mb-8 tw-w-72 tw-rounded-lg tw-text-white
+      dark:tw-text-darkGray-100
+      tw-py-4 tw-pb-4 tw-px-4
+      box
+    "
+  >
+    <div class="tw-text-xl tw-font-semibold tw-mb-2">{{ $t('store.currentEra') }}</div>
+    <div class="tw-flex tw-flex-col tw-items-center tw-mb-1">
+      <div class="tw-font-bold tw-text-xl">
+        {{ era }}
+      </div>
+    </div>
+    <div class="">
+      <div class="tw-font-semibold tw-mb-1">
+        {{ $t('store.blocksUntilNextEra') }}
+      </div>
+      <q-linear-progress size="22px" :value="progress" color="secondary" stripe rounded>
+        <div class="absolute-full flex flex-center">
```

Comment:
![Screen Shot 2021-11-22 at 10 13 09 AM](https://user-images.githubusercontent.com/40356749/142834189-8fb72339-0949-44ea-909a-4ab6c9c9e7bd.png)
The progress stripe should move or have some sort of visual indicator that the block number is changing

Reply:
Updated
https://gyazo.com/317327b2e392790ae5e52d07c3d79112